cmd/internal/register/maps.go

```diff
@@ -217,12 +211,12 @@ func Maps(a *atlas.Atlas, confMaps []config.Map, providers map[string]provider.T
 				}
 			}
 
-			var minZoom uint
+			minZoom := uint(1)
```

Comment:
This should be 0

Reply:
We should make MinZoom a const a la [MaxZoom](https://github.com/go-spatial/tegola/blob/573d26150c34970fcd54afabfbbc64f4a347609b/atlas/atlas.go#L21)